bgpd/bgp_attr.c

```diff
@@ -2755,6 +2758,38 @@ bgp_attr_parse_ret_t bgp_attr_parse(struct peer *peer, struct attr *attr,
 	return BGP_ATTR_PARSE_PROCEED;
 }
 
+/*
+ * Extract the tunnel type from extended community
+ */
+bool bgp_attr_extcom_tunnel_type(struct attr *attr,
```

Comment:
This change needs it's own commit.  The move around of bgp_attr_extcom_tunnel_type is one logical piece of work.  Please create a commit with just this change in it.

Reply:
additionally this function returns a bool but we never use it at all.  Why even return a bool?

Reply:
I thought, in future, if we ever needed to print a debug statement to figure out whether extcom had tunnel type attribute or not, the bool will come in handy..